mne_qt_browser/_pg_figure.py

```diff
@@ -2099,7 +2099,7 @@ def update_label_pos(self):
 
 class _AnnotEditDialog(_BaseDialog):
     def __init__(self, annot_dock):
-        super().__init__(annot_dock.main, title='Edit Annotations')
+        super().__init__(annot_dock.weakmain, title='Edit Annotations')
```

Comment:
I would expect it to need to be this to resolve to the object itself or to None
```suggestion
        super().__init__(annot_dock.weakmain(), title='Edit Annotations')
```

Reply:
:see_no_evil: Yes.. without it breaks the annotation edit.